frontend/src/pages/Dashboard.js

```diff
@@ -305,8 +343,22 @@ const Dashboard = () => {
                 <UploadFileModal
                     isOpen={isUploadModalOpen}
                     onClose={onCloseUploadModal}
-                    startingUploadedFiles={uploadedFiles}
+                    startingUploadedFiles={uploadedFiles?.filter(
+                        (file) => !file.is_supporting_document
+                    )}
+                    refetchFiles={getUploadedFiles}
+                    isSupportingDocument={false}
```

Comment:
I would prefer this as an optional parameter personally. assume it is not (default false)